app/models/document_capture_session.rb

```diff
@@ -52,7 +59,7 @@ def create_doc_auth_session
     save!
   end
 
-  def store_doc_auth_result(result:, pii:)
+  def store_doc_auth_result(result:, pii:) # IS THIS CALLED ANYWHERE?
```

Comment:
Looks like it's called from #set_up_document_capture_result in doc_auth_helper.rb ... which isn't called anywhere. It might be left over from removing the doc_auth async code.